### ISSUE TITLE
(non-issue) question about using jsdevlivr with esm.sh

### ISSUE THREAD

**Issue by ije** [boywithkeyboard-archive/cheetah1#65]
Hi, i'm the maintainer of esm.sh. This project looks promising! i'm trying to learn your code, i found something vary interesting and i‘m just wondering can you please share why you choosed jsdevlivr to import npm packages with esm.sh types. i'm not saying you should use esm.sh, jsdevlivr is great! i'm just curious did you try to use esm.sh with trouble or something, if that i can fix/improve.

> ```js
> // @deno-types='https://esm.sh/@sinclair/typebox@0.28.16/value'
> import { Value } from 'https://cdn.jsdelivr.net/npm/@sinclair/typebox@0.28.16/value/value.js/+esm'
> ```

anyway, thanks for creating this cool project! (and sorry i filed this _non-issue_ issue i did not find the GH discusisions enabled.)

**Comment by boywithkeyboard** [comment]
Hey @ije!

When I wrote this code, esm.sh seemed to inject polyfills for Node.js, even though those libraries were also built for the browser. So I ended up with a series of `Deno.` calls that caused the app to fail when deployed to Cloudflare Workers. I used jsDelivr because they are more reliable (they don't try to make a package to be compatible with Deno).

I've recently experimented with adding the query parameter ?target=es2022" in other projects, but I'm not sure how to avoid the "conversion from Node to Deno". This said, the issue might already be solved, but I'm not sure. TypeBox, on the other hand, is very complex and I decided to go that route as it seemed more reliable.

Don't get me wrong, esms.sh is great - I use it for a number of private codebases, but in this particular case it didn't seem to be the best choice.
